app/build.gradle

```diff
@@ -3,13 +3,14 @@ apply plugin: 'com.github.triplet.play'
 
 android {
     compileSdkVersion 28
+    buildToolsVersion '28.0.3'
```

Comment:
Not required. Without this it'll take latest one for compile sdk, which is what we need

Reply:
hmmk